packages/design-system/src/components/Icon/SizedIcon.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { HTMLAttributes } from 'react';
+import { Icon, IconSize } from '@talend/icons';
+
+const getNumericSize = (size: IconSize) => {
+	return {
+		XS: '0.8rem',
+		S: '1.2rem',
+		M: '1.6rem',
+		L: '2.4rem',
+	}[size];
+};
```

Comment:
Instead of duplicating the values, should we use sass with token usage ?

Reply:
We could